README.md

```diff
@@ -102,3 +112,49 @@ mnc2nii PATH_TO/avg.XXX.mnc PATH_TO/template_XXX.nii
 ~~~
 
 </details>
+
+### Steps to register a subject to the T2w dog template. 
+
+We picked `sub-HarshmanDobby` for this.
+
+#### Data
+
+**[with internal data access]** Required data is at the `data.neuro.polymtl.ca:datasets/template_dog_virginiatech` repository under the `rb/update_HB` branch
```

Comment:
open an issue that the link to a branch needs to be updated eventually

Reply:
Done in issue #23